src/services/vyos-hostsd

```diff
@@ -255,6 +263,13 @@ def handle_message(msg_json):
     with open(STATE_FILE, 'w') as f:
         json.dump(STATE, f)
 
+    if changes > 0:
+        (ret,return_code) = popen("pidof pdns_recursor");
```

Comment:
Why not use `vyos.util.process_named_running()` here?

Reply:
Sounds good to me.  Also found a bug (which python probably would ignored)